### PR TITLE
Adds info about allowed_contributors

### PR DESCRIPTION
**Pull request by radwo**

### CODE REVIEW

File: projects-yaml-reference_5b6813d20428631d7a89afa9.md [hunk 1]
```diff
@@ -123,26 +125,30 @@ List of values for `run`: `true`, `false`
...
-The `forked_pull_requests` property is used for holding the `allowed_secrets` property.
+The `forked_pull_requests` property is used for holding the `allowed_secrets`, and `allowed_contributors` properties.
 
 ###### allowed_secrets
 
 The `allowed_secrets` property specifies array of secrets names that are allowed to be exported into jobs triggered by `forked-pull-requests`. If the array is empty, no secret will be exported.
 
+###### allowed_contributors
+
+The `allowed_contributors` property specifies array of contributors GitHub logins that are allowed to run workflows triggered by `forked-pull-requests`. If the array is empty, all contributors are allowed.
```
Comment (shiroyasha) [inline projects-yaml-reference_5b6813d20428631d7a89afa9.md]: Keep it under 80 chars in length :)

File: project-workflow-trigger-options_5d42dcce042863478675bb99.md [hunk 2]
```diff
@@ -60,6 +61,11 @@ To distinguish workflows from main and forked repositories, you can compare
 By default Semaphore will not inject any secrets into jobs for pull requests from forks.
 You can whitelist the secrets you want to expose specifying their names.
 
+### Filter contributors in forked pull requests
+
+By default Semaphore will allow all contributors to create workflows for pull requests from forks.
+You can filter the contributors you want to allow to do that specifying their github logins.
```
Comment (shiroyasha) [inline project-workflow-trigger-options_5d42dcce042863478675bb99.md]: Run this through Grammarly. Some commas are missing.

---

The second sentence is hard to read, it is hard to reason what `that` is.

File: project-workflow-trigger-options_5d42dcce042863478675bb99.md [hunk 3]
```diff
@@ -45,6 +45,9 @@ When you choose this option Semaphore will create a workflow for every push to t
 In every job from this workflow, Semaphore will export
 `SEMAPHORE_GIT_REF_TYPE` environment variable with value `pull-request`.
 
+Semaphore will use the MERGE commit to run a workflow. The SHA of the HEAD commit 
+of the Pull Request will be stored in `SEMAPHORE\_GIT\_PR\_SHA` environment variable.
```
Comment (shiroyasha) [inline project-workflow-trigger-options_5d42dcce042863478675bb99.md]: Technical writing is almost exclusively in the simple present tense. Consider rewriting the sentence to remove the future tenses.

Example:

> Semaphore uses the MERGE commit to run a workflow. The SHA of the HEAD commit 
of the Pull Request is stored in `SEMAPHORE\_GIT\_PR\_SHA` environment variable.

**Comment by radwo** [comment]
Thanks for review and tips @shiroyasha ❤️

**Comment by markoa** [comment]
I've extended the [writing guidelines](https://github.com/semaphoreci/docs/blob/master/CONTRIBUTING.md#writing) with this feedback.

**Pull request merged**